p2p/eth.py

```diff
@@ -70,6 +73,31 @@ class GetBlockHeaders(Command):
     ]
 
 
+class HeaderRequest(BaseHeaderRequest):
+    MAX_HEADERS_FETCH = MAX_HEADERS_FETCH
+
+    def __init__(self,
+                 block_number_or_hash: BlockIdentifier,
+                 max_headers: int,
+                 skip: int,
+                 reverse: bool) -> None:
+        self.block_number_or_hash = block_number_or_hash
+        self.max_headers = max_headers
+        self.skip = skip
+        self.reverse = reverse
+
+    def validate_response(self, response: Any) -> None:
```

Comment:
Do we need to use `Any` here, even though you raise a ValidationError if it's not a tuple?

Reply:
I believe `Any` is what we want.  It is allowed to pass anything into this function, but it will raise a validation error if it's not the correct type or it isn't well formed.

That said, I can see an argument for dropping both `isinstance` checks here if we think that `mypy` is sufficient to catch those cases (Which is the way I'm leaning)